bus-mapping/src/evm.rs

```diff
@@ -347,15 +372,36 @@ mod evm_tests {
     }
 
     #[test]
-    fn evmword_from_str() -> Result<(), Error> {
+    fn evmword_from_str() {
         let word_str =
             "000000000000000000000000000000000000000000000000000c849c24f39248";
 
         let word_from_u128 = EvmWord::from(3523505890234952u128);
-        let word_from_str = EvmWord::from_str(word_str)?;
+        let word_from_str = EvmWord::from_str(word_str).unwrap();
 
         assert_eq!(word_from_u128, word_from_str);
-        Ok(())
+    }
+
+    #[test]
+    fn evmword_add() {
+        // Test add
+        let a = EvmWord::from_str("deadbeef").unwrap();
+        let b = EvmWord::from_str("faceb00c").unwrap();
+        assert_eq!(
+            a.adc(b).unwrap().to_hex(),
+            "00000000000000000000000000000000000000000000000000000001d97c6efb"
+        );
+
+        // Test add Error
+        let c = EvmWord::from_str(
+            "8000000000000000000000000000000000000000000000000000000000000000",
+        )
+        .unwrap();
+        let d = EvmWord::from_str(
+            "8000000000000000000000000000000000000000000000000000000000000000",
+        )
+        .unwrap();
+        assert_eq!(&format!("{:?}", c.adc(d)), "Err(EvmWordAddingOverflow)");
```

Comment:
Can we make the test return `Result<(), Error>` So that we can use `?` instead of `unwrap` inside of the test in as many places as possible?
That will make the errors easier to debug.

Reply:
> That will make the errors easier to debug.
I understand it!